README.md

```diff
@@ -51,9 +51,12 @@ JoularJX can be configured by modifying the ```config.properties``` files:
 - ```overwrite-runtime-data```: overwrite runtime power data files, or if set to false, it will write new files for each monitoring cycle.
 - ```logger-level```: set the level of information (by logger) given by JoularJX in the terminal (allowed values: OFF, INFO, WARNING, SEVERE).
 - ```powermonitor-path```: Full path to the power monitor program (only for Windows).
-- ```track-consumption-evolution```: generate .csv files for each method containing details of the method's consumption over the time. Each consumption value is mapped to an Unix timestamp.
+- ```track-consumption-evolution```: generate CSV files for each method containing details of the method's consumption over the time. Each consumption value is mapped to an Unix timestamp.
 - ```evolution-data-path```: path to the location where the consumption evolution csv files will be stored. The tool will attemp to create the folders if they do not alreay exists.
 - ```hide-agent-consumption```: if set to true, the energy consumption of the agent threads will not be reported.
+- ```enable-call-trees-consumption```: compute and save method call trees energy consumption. The data will be stored in CSV files.
+- ```save-call-trees-runtime-data```: write runtime call trees power consumption in a CSV file.
```

Comment:
Clarify that it is on every runtime cycle (1 second), and filename with timestamp